tdclient/client.py

```diff
@@ -459,6 +460,24 @@ def bulk_imports(self):
         """
         return [ models.BulkImport(self, **data) for data in self.api.list_bulk_imports() ]
 
+    @staticmethod
+    def validate_part_name(part_name):
```

Comment:
It'd be better to move this validation code into `tdclient.bulk_import_api.BulkImportAPI` class because `tdclient.client.Client` class is just a wrapper to `tdclient.bulk_import_model.BulkImport` to do the validation on all BulkImport API calls.

https://github.com/treasure-data/td-client-python/blob/49533202533a1a97539e189625537dd654a96dc3/tdclient/bulk_import_api.py#L16